### PR TITLE
UTs for ConvertProperties

### LINKED ISSUE #1325: Test failing after #1323 got merged, and reverted by #1324 needs to be fixed
#1324 removed a test failing due to #1323, but it's unclear if that test is bad or if there's an actual bug.

### PR DESCRIPTION
**Pull request by himanshug**
fixes #1325

### CODE REVIEW

File: services/src/test/java/io/druid/cli/convert/ConvertPropertiesTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    Properties expectedConvertedProperties = new Properties();
+    expectedConvertedProperties.load(this.getClass().getResourceAsStream("/convertProps/new.properties"));
+
+    Assert.assertEquals(expectedConvertedProperties, actualConvertedProperties);
+  }
+
+  @After
+  public void tearDown()
+  {
+    tmpFolder.delete();
```
Comment (drcrallen) [inline services/src/test/java/io/druid/cli/convert/ConvertPropertiesTest.java]: Will not delete directory. You should `FileUtils.deleteDirectory`

Comment (himanshug) [inline services/src/test/java/io/druid/cli/convert/ConvertPropertiesTest.java]: tmp folder does get deleted , I checked.
https://github.com/junit-team/junit/wiki/Rules#temporaryfolder-rule

In fact this @After tearDown() isn't really required, deletion is part of the rule.

however, it won't raise any error if deletion wasn't successful for some reason, so putting FileUtils.deleteDirectory to be really really sure about it.

File: services/src/main/java/io/druid/cli/convert/ConvertProperties.java [hunk 2]
```diff
@@ -68,7 +68,16 @@ private static String parseJdbcUrl(String jdbcUrl){
       new Rename("druid.db.connector.connectURI", "druid.metadata.storage.connector.connectURI"),
       new Rename("druid.db.connector.user", "druid.metadata.storage.connector.user"),
       new Rename("druid.db.connector.password", "druid.metadata.storage.connector.password"),
+      new Rename("druid.db.connector.validationQuery", "druid.metadata.storage.connector.validationQuery"),
+      new Rename("druid.db.connector.useValidationQuery", "druid.metadata.storage.connector.useValidationQuery"),
```
Comment (xvrl) [inline services/src/main/java/io/druid/cli/convert/ConvertProperties.java]: Validation query configs should be removed. It's hard coded in the storage module and always used by default in 0.7

Comment (himanshug) [inline services/src/main/java/io/druid/cli/convert/ConvertProperties.java]: removed.

**Pull request merged**